kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -449,10 +449,22 @@ private void handleModemManagerGPSSetup(Device device, Optional<Boolean> enableG
         Properties modemLocationProperties = this.dbusConnection.getRemoteObject(MM_BUS_NAME,
                 modemLocation.getObjectPath(), Properties.class);
 
-        Set<MMModemLocationSource> availableLocationSources = MMModemLocationSource
-                .toMMModemLocationSourceFromBitMask(modemLocationProperties.Get(MM_LOCATION_BUS_NAME, "Capabilities"));
-        Set<MMModemLocationSource> currentLocationSources = MMModemLocationSource
-                .toMMModemLocationSourceFromBitMask(modemLocationProperties.Get(MM_LOCATION_BUS_NAME, "Enabled"));
+        Set<MMModemLocationSource> availableLocationSources = EnumSet
+                .of(MMModemLocationSource.MM_MODEM_LOCATION_SOURCE_NONE);
+        Set<MMModemLocationSource> currentLocationSources = EnumSet
+                .of(MMModemLocationSource.MM_MODEM_LOCATION_SOURCE_NONE);
+
+        try {
+            availableLocationSources = MMModemLocationSource.toMMModemLocationSourceFromBitMask(
+                    modemLocationProperties.Get(MM_LOCATION_BUS_NAME, "Capabilities"));
+            currentLocationSources = MMModemLocationSource
+                    .toMMModemLocationSourceFromBitMask(modemLocationProperties.Get(MM_LOCATION_BUS_NAME, "Enabled"));
+        } catch (DBusExecutionException e) {
+            logger.debug("Cannot retrive Modem.Location capabilities for {}. Caused by: ",
+                    modemLocationProperties.getObjectPath(), e);
```

Comment:
I was in doubt about this. It can be missing because the interface is not yet ready, or because the Modem doesn't support the Location interface... it is not necessarily an error.

I can change it to `warning` if you want. I looks like it is more consistent with the other similar log entries in the same file.

Reply:
Fixed in: 186be0ffbe630c90deee6487d25d136fc5ff4a96

Reply:
I supposed that this could be not an error, but I agree that a warning is more appropriate.